Release 1.0.0

Quantumplation opened the pull request.

SupernaviX commented:
LGTM

OmarDarwish commented on README.md:
````diff
@@ -73,6 +73,20 @@ logger.context.currentBlock = 'Phase3';
+the the isCapillaryCompatible property on the logger, as so:
+
+```javascript
+if(parentLogger && parentLogger.isCapillaryCompatible) {
````
Space after `if` #nitpick

OmarDarwish reviewed (approved):
LGTM

Quantumplation committed:
[docs] Update README.md

Quantumplation committed:
[build] Bump the version to 1.0.0

Quantumplation committed:
[dev] Add an isCapillaryCompatible flag

This flag will allow other module writers to safely detect whether they can leverage capillary features

Quantumplation committed:
[build] Export the NullLogger for external use
